WordPress/Docs/Arrays/MultipleStatementAlignmentStandard.xml

```diff
@@ -0,0 +1,44 @@
+<documentation title="Multiple Statement Alignment">
+    <standard>
+    <![CDATA[
+      When declaring arrays, there should be one space on either side of a double arrow operator used to assign a value to a key.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: correct spacing between the key and value.">
+        <![CDATA[
+$args( 'cat'<em> => </em>22, 'year'<em> => </em>$current_year );
```

Comment:
Multi-item associative arrays are not allowed to be single-line according to the WP Coding Standards, so while this sniff doesn't address that, the example does not comply with the other coding standards.

It also isn't an array declaration the way it is now (it's actually a parse error).

May I suggest splitting this and having two code samples for both `valid` as well as `invalid` for this block ?

```xml
        <code title="Valid: correct spacing between the key and value.">
        <![CDATA[
$foo = array( 'cat'<em> => </em>22 );
$bar = array( 'year'<em> => </em>$current_year );
        ]]>
        </code>
        <code title="Invalid: No or incorrect spacing between the key and value.">
        <![CDATA[
$foo = array( 'cat'<em>=></em>22 );
$bar = array( 'year'<em>=>   </em>$current_year );
        ]]>
        </code>
```